examples/psyir/README.md

```diff
@@ -63,3 +63,14 @@ To run this example:
 ```sh
 > python create_aggregate_types.py
 ```
+
+## Example 3:
+
+Demonstrates how to manipulate an existing PSyIR tree. This example
+imports the tree already created in Example 1 and applies some modifications
```

Comment:
imports the PSyIR created in Example 1 and writes it out in Fortran. It then applies some modifications to the imported PSyIR and writes out the modified PSyIR in Fortran, allowing the differences to be easily seen. This example may be run ...

Reply:
Well I removed the "writing out from example 1" now, so I am not modifying the text as suggested, but tell me if this is something you want me to add again.